DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsController.cs

```diff
@@ -387,7 +387,7 @@ private IActionResult EditRegistrationPromptPostSave(EditRegistrationPromptViewM
 
         private IActionResult AddRegistrationPromptConfigureAnswersPostNext(RegistrationPromptAnswersViewModel model)
         {
-            IgnoreAddNewAnswerValidation();
+            ModelState.ClearAllErrors();

```

Comment:
Not your change but is the if-statement below ever true when we clear the errors?

Reply:
I think not. Both this method and EditRegistrationPromptPostSave previously called IgnoreAddNewAnswerValidation before checking ModelState.IsValid which does exactly the same thing as ModelState.ClearAllErrors. I think they might have been holdovers that were never removed, so I'll remove them now. But if they are necessary for some bizarre reason, it will be easy enough to add them back.